exercises/yacht/src/Yacht.hs

```diff
@@ -0,0 +1,17 @@
+module Yacht (yacht, Category(..)) where
+
+data Category = Ones
+              | Twos
+              | Threes
+              | Fours
+              | Fives
+              | Sixes
+              | FullHouse
+              | FourOfAKind
+              | LittleStraight
+              | BigStraight
+              | Choice
+              | Yacht
+
+yacht :: Category -> [Int] -> Int
+yacht = error "You need to implement this function."
```

Comment:
I thought we should provide a name for the two function parameters, such as `yacht category dice = error "You need to implement this function.". this is based on `grep error exercises/*/src/*.hs` - they all provide names. I think I was not clear in my previous comment. I am sorry for not being clear.

Reply:
I thought we should provide a name for the two function parameters, such as `yacht category dice = error "You need to implement this function."` this is based on `grep error exercises/*/src/*.hs` - they all provide names. I think I was not clear in my previous comment. I am sorry for not being clear.